src/Domain/Donation.php

```diff
@@ -562,6 +568,15 @@ public function toFrontEndApiModel(): array
             $data['matchReservedAmount'] = (float) $this->getFundingWithdrawalTotal();
         }
 
+        if ($this->mandate) {
+            // Not including the entire mandate details as that would be wasteful, just parts FE needs to display with
+            // the donation.
+            $data['mandate']['uuid'] = $this->mandate->getUuid()->toString();
```

Comment:
This will cause an extra db query to load each mandate. Not ideal and we could eagerly load them to stop that but probably not an issue for the current feature as one donor is never going likely to have hundreds of mandates.

Might be a problem for things that deal with donations more in bulk though in a future when we have hundreds or thousands of mandates in the database.